### ISSUE TITLE
GA4 Ecommerce?

### ISSUE THREAD

**Issue by kevinvermeulenn** [OFFLINE-GmbH/oc-mall-plugin#926]
Is this plugin ready for Google Analytics 4 with the predefined events?

**Comment by PubliAlex** [comment]
I don't think so, as everything with october, as you have total control over the template, you can add them by yourself

**Comment by tobias-kuendig** [comment]
This is currently untested and I do not have any knowledge about GA4 in particular, so I'm probably not the best person to build this feature. But as @PubliAlex mentioned, customizing the partials can go a long way.

PRs from the community are of course welcome.

**Comment by damsfx** [comment]
@tobias-kuendig I'm continuing to integrate GA4, but I'm encountering a few difficulties.

When I listen to the `mall.wishlist.productAdded` event, according to the documentation :
> This component emits a `mall.wishlist.productAdded` JavaScript event when 
a product was added to a wishlist. You can subscribe to this event
and receive the added product's information.

But the data passed to the event doesn't seem to contain any information about the product added to the wishlist.
